src/backend/distributed/commands/multi_copy.c

```diff
@@ -829,15 +937,6 @@ OpenCopyConnections(CopyStmt *copyStatement, ShardConnections *shardConnections,
 		StringInfo copyCommand = NULL;
 		PGresult *result = NULL;
 
-		/*
```

Comment:
I think we should rename this function and update the function comment. I'd even add an `assert (placement->partitionMethod != DISTRIBUTE_BY_HASH)`. Otherwise it is becoming confusing to have two different ways of handling connection opening.

Maybe `OpenCopyConnectionsForNewShards`

Reply:
done